app/services/course/assessment/question/programming_import_service.rb

```diff
@@ -95,6 +95,8 @@ def build_template_file_records(template_files)
   # @return [Array<Course::Assessment::Question::ProgrammingTestCase>]
   def build_test_case_records(test_report)
     test_cases = parse_test_report(test_report)
+    # Sort test cases lexicographically to run tests in order by test types
```

Comment:
The test case records here should have no effect on the order in which the tests are run. It so happens that Python's unit test framework runs tests in lexicographical order, this might not be true of other unit test frameworks.

I think what this actually does is to cause the test case records to be inserted into the db in order, which causes them to appear in order when viewed. Can you verify this and update the comment?

Also, if the above is true, sort at the controller or view when the test cases are being displayed instead of relying on them being retrieved in order of ID, as that's not always the case in production.